tools/build/Makefile.dev

```diff
@@ -0,0 +1,15 @@
+# This serves for the local development only. After the first building by 
```

Comment:
Will this help with docker build or only standalone build ?

Reply:
I don't understand what the standalone build is ~

The intention is to help such a scenario that updates codes and then rebuilds for testing locally. In such scenario, the initial setup (including upgrading tools such `pip`) for `virtualenv` should work all along and requires no redo every time, and we just need to run the `${TCF_HOME}/scripts/rebuild.sh` to rebuild the whole project

Reply:
By standalone build, I mean build performed outside docker (i.e not via docker-compose up). I'm thinking would it make more sense to move the pip install packages in tools/build/Makefile to Dockerfile and make it part of docker image (i.e use apt-install of python pip packages)

Reply:
Gotcha~ 

I found that running `make packages` work equivalently with this PR, so it's closed.

Moving the virtualenv setup into Dockerfile is a good . I'll work on that later.